src/components/Comments.jsx

```diff
@@ -15,10 +19,45 @@ export const Comments = ({ singleReview }) => {
     });
   }, [review_id]);
 
+  const handleReviewComment = (e) => {
+    e.preventDefault();
+    setErr(null);
+    buttonRef.current.disabled = true;
+    postReviewComment(review_id, newReviewComment)
+      .then((newComment) => {
+        setComments((currComments) => {
+          return [newComment, ...currComments];
+        });
```

Comment:
When the posting has happened it would be great to un-disable the submit button so the user can use the form again after their first comment has been stored

Reply:
It would also be great if the value of the input form is reset once the comment has been submitted so the form clears